### DISCUSSION TITLE
Integrations

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [trpc/trpc#4082] [Ideas]
I'm learning Nuxt 3, Zod, and tRPC. This package worked great in my learning https://github.com/wobsoriano/trpc-nuxt.

So when I went to the tRPC website, I noticed "Using Next.js" in the top toolbar...

Can this section be renamed to "Integrations", and "Using Nuxt" added to it?

Nice project! Wishing you all health, peace, and success.

**Comment by Nick-Lucas** [comment]
I love this idea. We have an ongoing major project #3963 which this would fall under as it will take some new pages, creating a space to list community integrations is a great addition 

Would you post your suggestion in that thread?
